modules/lymphgen/1.0/schemas/base-1.0.yaml

```diff
@@ -0,0 +1 @@
+../../../../schemas/base/base-1.0.yaml
```

Comment:
Since this doesn't use the samples table, it's probably a good idea to create a new schema for this module that doesn't make _any_ requirements of the samples table and remove this one. Otherwise you can get annoying errors about missing stuff.